Source/UI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -16,7 +16,7 @@ protocol BillingFormTextFieldDelegate: AnyObject {
 protocol BillingFormViewControllerDelegate: AnyObject {
     func doneButtonIsPressed(sender: UIViewController)
     func cancelButtonIsPressed(sender: UIViewController)
-    func getViewForHeader(sender: UIViewController) -> UIView?
+    func getViewForHeader() -> UIView?
```

Comment:
I've made a separate comment focussing on naming of this method. Now I also wonder whether we need this method at all.

Its purpose seems to be enabling ViewModel to "create an UIView". This blurs the lines between UI and non UI code. Normally the point of a ViewModel is to cover business logic, and keep UI logic into the VC. As such, any UI component creation should be the responsibility of the VC and not the ViewModel.

I'd suggest we remove this method and move the current `getViewForHeader` from `DefaultBillingFormViewModel` to `BillingFormViewController`.